### PR TITLE
ADPT-161:  Renamed tests according to guidelines

### PR DESCRIPTION
**Pull request by ensarevlideveloper**
Renamed tests according to coding guidelines. Introduce Architecture Test checking name conventions in tests.

[Sonar](https://sonarcloud.io/summary/new_code?id=ensarevlideveloper_TaskanaAdapter&branch=ADPT-161)
---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [X] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [X] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [X] I put the ticket in review

### Verified by the reviewer:
- [ ] Commit message format → ADPT-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/ArchitectureTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  @BeforeAll
+  static void init() {
+    // time intensive operation should only be done once
+    importedClasses =
+        new ClassFileImporter()
+            .importPackages(
+                "pro.taskana.adapter.integration",
+                "pro.taskana.adapter.systemconnector.api",
+                "pro.taskana.adapter.camunda.dto",
+                "pro.taskana.camunda.camundasystemconnector.acceptance");
```
Comment (mustaphazorgati) [inline taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/ArchitectureTest.java:27]: why not only `pro.taskana`?

**Review by mustaphazorgati — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 2]
```diff
@@ -1,109 +1,161 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>pro.taskana</groupId>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (mustaphazorgati) [inline pom.xml:1]: What happened here? Is your formatter set up properly?

File: taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/ArchitectureTest.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import org.junit.jupiter.api.TestTemplate;
+
+class ArchitectureTest {
+
+  private static JavaClasses importedClasses;
+
+  @BeforeAll
+  static void init() {
+    // time intensive operation should only be done once
+    importedClasses = new ClassFileImporter().importPackages("pro.taskana");
```
Comment (mustaphazorgati) [inline taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/ArchitectureTest.java:21]: Please import the `accaptance` package aswell, since we have 2 Files which are in an `acaptance..` package.

**Review by mustaphazorgati — CHANGES_REQUESTED** [review]

**Review by mustaphazorgati — APPROVED** [review]
